maas/resource_maas_instance.go

```diff
@@ -35,13 +35,18 @@ func resourceMaasInstance() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			"allocate_params": {
-				Type:        schema.TypeSet,
```

Comment:
Same thing applies for `deploy_params` and also the same fields of `block_device` resource

Reply:
Thats why I split each linter check fix into it's own commit and wanted to rebase instead of squashing.
This change comes from this commit https://github.com/maas/terraform-provider-maas/pull/102/commits/1ce95f982d55afdfcd01adf6a4573ff35956a317 as [S018](https://github.com/bflad/tfproviderlint/tree/main/passes/S018) suggests `List` over `Set` if `MaxItems` is `1`